ocaml/xen-api-client/lib_test/xen_api_test.ml

```diff
@@ -88,6 +88,12 @@ module Fake_IO = struct
     connections := c :: !connections ;
     return (Ok (ic, oc))
 
+  let close (ic, oc) =
+    let this_one c = c.ic == ic && c.oc == oc in
```

Comment:
this has two usages of `==` but you are asserting one ..?

Reply:
it's one line